include/data.h

```diff
@@ -88,7 +88,7 @@ struct Trainer
     /*0x13*/ u8 trainerName[TRAINER_NAME_LENGTH + 1];
     /*0x1E*/ bool8 doubleBattle:1;
              bool8 mugshotEnabled:1;
-             u8 padding:6;
+             u8 startingStatus:6;    // this trainer starts a battle with a given status. see include/constants/battle.h for values
```

Comment:
I assume this doesn't allow for stacking effects, e.g. Tailwind on both sides or Trick Room and a terrain?

Reply:
if you add two effects, none of them will work.